packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -374,15 +374,27 @@ export class StripeHelper extends StripeHelperBase {
     email: string,
     displayName: string,
     idempotencyKey: string,
-    ipAddress?: string
+    countryCode?: string,
+    postalCode?: string
```

Comment:
Could make this require both of them, or neither of them with an optional union type perhaps?

This is getting long enough where it'd make sense later to change it to taking an object. There's a VS Code refactor that will switch it, but cause the tests aren't TS, it can't auto-fix those. 😢